### DISCUSSION TITLE
Why no Nuclei

### DISCUSSION THREAD

**Discussion by AkikoOrenji** [PortSwigger/BChecks#68] [General]
Perhaps a dumb question but why wasn't this feature built to work with Nuclei templates?  Apart from being integrated directly with burp do BChecks provide functionality that Nuclei doesn't ? 

It seems like with so many Nuclei templates out there it would make sense to leverage all that work.

**Comment by A-J-C** [comment]
One of our developers had this to say on Twitter around a similar question: https://twitter.com/TomLefley/status/1668256067015397376

**Reply by AkikoOrenji** [reply]
hadn't seen that so thanks.
